source/_docs/scripts/service-calls.markdown

```diff
@@ -28,7 +28,7 @@ Instead of targeting an entity, you can also target an {% term area %} or {% ter
 This is done with the `target` key.
 
 A `target` is a map that contains at least one of the following: `area_id`, `device_id`, `entity_id`.
-Each of these can be a list.
+Each of these can be a list. The values of `entity_id` are case–insensitive, while `area_id` and `device_id` are case–sensitive.
```

Comment:
Everything should be considered case-sentitive. Let's adjust the text to reflect that.

Reply:
Actually, when I'm trying to create zones `foo` and `FOO` at http://homeassistant.local:8123/config/areas/dashboard -> I'm getting `The name FOO (foo) is already in use` error.

That would mean, that the zones should also be case–**in**sensitive when being a target. Or the other way, id they _should_ be case–sensitive, that would mean that there is a bug in the dashboard…

Reply:
> That would mean, that the zones should also be case–insensitive when being a target. 

You are mixing up scopes.

> Anyway: I'd love it if the docs reflected the real behaviour, not the wishful thinking.

Yup, well, there is a backward compatibility layer active, but entities IDs have naming limits and are case sensitive (capital letters cannot be used in entity names, try it yourself, rename an entity ID).

Reply:
> You are mixing up scopes.

Kinda, probably, yeah. I'm looking for consistency -> some way to lower [WTF–factor](https://www.reddit.com/r/ProgrammerHumor/comments/cd1xio/wtfs_per_minute/) ;-)

> capital letters cannot be used in entity names, try it yourself, rename an entity ID

When I edit entity_id with changed caps, the error is that the entity_id already exists -> proving in yet another place, that entity_ids are case–insensitive, not case–sensitive.

The error message does not state that capital letters are invalid:

![Screenshot 2024-02-06 at 23 16 24](https://github.com/home-assistant/home-assistant.io/assets/136261/a0557d45-a11e-498f-8b41-1a09be1bf04d)

Reply:
Still, they are consider so. The point is: We have some backward compatible logic/history in place. That doesn't make it a good idea or feature.

Please adjust the text as suggested, thanks!

Reply:
> Everything should be considered case-sentitive.

I'm not sure how to phrase that… Maybe just say the values should be lower-cased (if those are the actual rules for IDs)?

I mean, if the letters cannot be uppercase, then the point about case–sensitivity is a bit moot.

Reply:
Maybe this?

```
Each of these can be a list. As always, be mindful about values [case sensitivity](/docs/configuration/yaml/#upper-and-lower-case).
```

Is there a better way to make the link?

Reply:
> I'm not sure how to phrase that… Maybe just say the values should be lower-cased

Perfect! 👌

Reply:
> > I'm not sure how to phrase that… Maybe just say the values should be lower-cased
> 
> Perfect! 👌

Rebased and pushed a new commit.